main.py

```
@@ -1,44 +1,412 @@
+            user_info['plan_expiry'] = None
+        
+        save_user_data(target_user, user_info)
+        
+        # Notify admin
+        bot.send_message(message.chat.id, f"✅ **User #{target_user}** upgraded: {old_plan.upper()} → {new_plan.upper()}")
+        
+        # Notify user
+        plan_emoji = {'free': '⚡', 'silver': '⭐', 'gold': '👑'}
+        try:
+            bot.send_message(target_user, f"🎉 **Plan Upgraded!**\n\nYour plan: {plan_emoji[new_plan]} **{new_plan.upper()}**\n\nEnjoy your new features!")
+        except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)